python/graphstorm/model/lm_embed.py

```diff
@@ -45,7 +47,7 @@ def update_bert_cache(g, lm_models_info, lm_models, lm_emb_cache, lm_infer_batch
         lm_model.eval()
         for ntype in lm_ntypes:
             if get_rank() == 0:
-                print('compute bert embedding on node {}'.format(ntype))
+                logging.debug('compute bert embedding on node %d', ntype)
```

Comment:
I would sugguest we use logging.info here. Otherwise, the system will stuck in computing bert embeddings without printing any information.